src/lib/rust.ts

```diff
@@ -285,9 +285,17 @@ const tryDownloadingBinary = async ({
 
   const url = `https://github.com/fluencelabs/marine/releases/download/${name}-v${version}/${name}-${platformToUse}-x86_64`;
 
+  startSpinner(
+    `Trying to download pre-built ${name}@${version} binary to ${replaceHomeDir(
+      dependencyDirPath
+    )}`
+  );
+
   try {
     await downloadFile(binaryPath, url);
+    stopSpinner();
   } catch {
+    stopSpinner("failed");
     return `Failed to download ${name}@${version} from ${url}`;
```

Comment:
Why no error in the message? `catch` should have caught some exception, right?

Reply:
good catch. Will add